package.json

```diff
@@ -14,6 +14,9 @@
   "bin": {
     "csharp-models-to-typescript": "./index.js"
   },
+  "scripts": {
+	  "test": "node index.js --config=./test-files/test-config.json && node ./test-files/test-file-exists.js --config=./test-files/test-config.json"
```

Comment:
I don't think this part `node ./test-files/test-file-exists.js --config=./test-files/test-config.json` is necessary and the file `test-file-exists.js`.
I'm confused on why we need to check the existence of a test-configuration in a test itself.

Reply:
I needed to check if the file exists to cover 2 cases:
1. When it does not find a .cs file to process, no error occurs and the output file is not created.
2. In some cases when conversion error occurs it creates the file and does not propagate the conversion error to the test, leaving the file empty in the output.

Reply:
I understand, thanks for the clarification.
Also, would it be possible to remove any existing `api.d.ts` file before running the test? So, that the test is repeatable in persisted environments (e.g. in locally)

Reply:
I tried to put `rm` before other test commands.
```shell
rm -f ./test-files/api.d.ts && node index.js --config=./test-files/test-config.json && node ./test-files/test-file-exists.js --config=./test-files/test-config.json"
```
But it will work only when running on Linux.
And I was too lazy to create a javascript just to delete the file.

Any ideia on how to do this?

Reply:
Yes. You could you `rimraf` library (save as dev dependency).
Then:   
`"test": "rimraf ./test-files/api.d.ts && node index.js --config=./test-files/test-config.json && node ./test-files/test-file-exists.js --config=./test-files/test-config.json"`

Reply:
It worked! Thanks!
Project updated.